### DISCUSSION TITLE
Best way to update transitive dependencies

### DISCUSSION THREAD

**Discussion by calto001** [renovatebot/renovate#29176] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab

### Please tell us more about your question or problem

I'm working on trying to keep transitive dependencies in node up to date. I began using `lockFileMaintenance` but I've noticed that if it's on the same schedule as the updates for my main dependencies then they will both try to update the same main packages, due to being in the semver range. Additionally if I have `minimumReleaseAge` set, `lockFileMaintenance` doesn't seem to respect that. 

So I was curious if there is a particular setting or workflow I should follow follow in order to be able to update only transitive  dependencies and not main packages. I know if I pinned the main dependencies in the `package.json` then this issue would be resolved, but I'm not sure if thats the only solution. I'm fairly new to this, so if I've missed something trivial please forgive me. 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Yes, you'd need to pin dependencies: https://docs.renovatebot.com/dependency-pinning/

**Reply by calto001** [reply]
Thank you @rarkins. Thats what I figured but wasn't 100% sure.
